src/components/Header.tsx

```diff
@@ -0,0 +1,29 @@
+import logo from "assets/img/logo.png";
+
+const Header = () => {
+  return (
+    <header className="sticky top-0">
+      <section className="mx-auto flex max-w-screen-lg flex-row flex-wrap items-center justify-between px-4 py-3">
```

Comment:
max-w-screen no need for this until your content is overflowing

Reply:
don't want the contents to go out beyond this max-w-screen-lg boundary, just setting the limit for vary wider screens